superset/assets/javascripts/chart/Chart.jsx

```diff
@@ -0,0 +1,115 @@
+/* eslint camelcase: 0 */
```

Comment:
I like the folder structure here with everything chart-related under a `chart/` folder